packages/server/src/integrations/tests/utils/postgres.ts

```diff
@@ -1,28 +1,23 @@
 import { Datasource, SourceName } from "@budibase/types"
-import { GenericContainer, Wait, StartedTestContainer } from "testcontainers"
-import env from "../../../environment"
+import {
+  PostgreSqlContainer,
+  StartedPostgreSqlContainer,
+} from "@testcontainers/postgresql"
 
-let container: StartedTestContainer | undefined
-
-const isMac = process.platform === "darwin"
+let container: StartedPostgreSqlContainer | undefined
 
 export async function getDsConfig(): Promise<Datasource> {
   try {
     if (!container) {
-      // postgres 15-bullseye safer bet on Linux
-      const version = isMac ? undefined : "15-bullseye"
-      container = await new GenericContainer("postgres", version)
+      const pgContainer = new PostgreSqlContainer()
```

Comment:
We probably want to be explicit with the version we use here, by default it appears to be Postgres 13: https://github.com/testcontainers/testcontainers-node/blob/cce5bccd7f4dee04de3948bdf7dae52e21969883/packages/modules/postgresql/src/postgresql-container.ts#L10.